src/components/App.js

```diff
@@ -17,8 +17,7 @@ import todosList from 'data'
 const App = () => {
   const [todos, setTodos] = useState(todosList)
   const [input, setInput] = useState('')
-
-  const [copiaTodos, setCopiaTodos] = useState(todos)
+  const [filterTodos, setFilterTodos] = useState('all')
```

Comment:
Para hacer el nombre más específico, podríamos ponerle statusFilter. De esa forma, queda bien en claro a qué filtro refiere, y no nos entra en conflicto con futuros filtros 😉.

```suggestion
  const [statusFilter, setStatusFilter] = useState('all')
```